File: MiniDOMAsteriskLib/CallManagers/Action/Logoff.cs
using minidom.CallManagers.Responses;

namespace minidom.CallManagers.Actions
{

    /// <summary>
    /// Effettua il logout
    /// </summary>
    /// <remarks></remarks>
    public class Logoff : Action
    {
        private string m_Username;
        private string m_Secret;

        public Logoff() : base("Logoff")
        {
        }

        protected override ActionResponse AllocateResponse(string[] rows)
        {
            return new LogoffResponse(this);
        }
    }
}

File: MiniDOMAsteriskLib/CallManagers/Action/Logoff.vb
Imports System.Net.Sockets
Imports System.Threading
Imports System.Net
Imports minidom.CallManagers.Responses

Namespace CallManagers.Actions

    ''' <summary>
    ''' Effettua il logout
    ''' </summary>
    ''' <remarks></remarks>
    Public Class Logoff
        Inherits Action

        Private m_Username As String
        Private m_Secret As String

        Public Sub New()
            MyBase.New("Logoff")
        End Sub

        

        Protected Overrides Function AllocateResponse(rows() As String) As ActionResponse
            Return New LogoffResponse(Me)
        End Function

    End Class

End Namespace